src/components/MusicSheet.vue

```diff
@@ -128,7 +128,10 @@ let noteTopPos = (function () {
 })();
 
 export default {
-  mounted: () => { $('#loginModal').modal('show') },
+  mounted: function () {
+    // login function will handle whether to call login modal
+    this.login()
```

Comment:
is this for persistence in the browser?

Reply:
Yup, that function will determine whether the user has been authenticated with a JWT and if not it'll open the Login modal for username/pword entry.